### PR TITLE
add replica field for fortio deployment

### PR DESCRIPTION
**Pull request by carolynhu**
We need to have this field to scale server and client side pods.

### CODE REVIEW

File: perf/benchmark/templates/fortio.yaml [hunk 1]
```diff
@@ -73,6 +73,11 @@ metadata:
     app: {{ $.name }}
   name: {{ $.name }}
 spec:
+{{- if eq $.name "fortioclient" }}
+replicas: 4
```
Comment (howardjohn) [inline perf/benchmark/templates/fortio.yaml]: why hardcoded to 4 and 20 instead of just `$.replicas` or something?

Comment (carolynhu) [reply perf/benchmark/templates/fortio.yaml]: just a wip pr, updated!

**Comment by carolynhu** [comment]
@howardjohn PTAL, thank!

File: perf/benchmark/setup_test.sh [hunk 2]
```diff
@@ -50,10 +52,12 @@ function run_test() {
       --set namespace="${NAMESPACE}" \
       --set excludeOutboundIPRanges=$(pod_ip_range)\
       --set includeOutboundIPRanges=$(svc_ip_range) \
-      --set client.inject="${ISTIO_INJECT}" \
+      --set server.replica="${SERVER_REPLICA}" \
+      --ser client.replica="${CLIENT_REPLICA}" \
```
Comment (richardwxn) [inline perf/benchmark/setup_test.sh]: this is a typo for "--ser", plz fix. @carolynhu

**Pull request merged**